numpy/_core/include/numpy/utils.h

```diff
@@ -36,15 +36,13 @@
 #define NPY_CAT_(a, b) NPY_CAT__(a, b)
 #define NPY_CAT(a, b) NPY_CAT_(a, b)
 
-static char get_installed_numpy_major_version()
+static PyObject *_npy_import_numpy_multiarray_umath()
 {
-    PyObject *numpy = PyImport_ImportModule("numpy");
-    PyObject *version_obj = PyObject_GetAttrString(numpy, "__version__");
-    Py_DECREF(numpy);
-    char major_version = PyUnicode_ReadChar(version_obj, 0);
-    Py_DECREF(version_obj);
-
-    return major_version;
+    PyObject *multiarray = PyImport_ImportModule("numpy._core._multiarray_umath");
+    if (multiarray == NULL) {
+        multiarray = PyImport_ImportModule("numpy.core._multiarray_umath");
```

Comment:
That is a try/except and somewhat legal, but not as is, you need to add a `PyErr_Clear()`, could check which exception type `PyErr_ExceptionMatches` (to be the same)

Reply:
Sure! I updated it.